### DISCUSSION TITLE
How to get the number of lines of a file?

### DISCUSSION THREAD

**Discussion by divramod** [MunifTanjim/nougat.nvim#73] [Q&A] [answered]
i saw this snippet in the examples:

```lua
stl:add_item(Item({
  hl = {fg = color.bg, bg = color.fg0},
  prefix = ' ',
  suffix = ' ',
  content = core.group({core.code('l'), ':', core.code('c')})
}))
```

is it somehow possible to get the number of lines a file has too with core.code?

i could not find the documentation for it.

**Comment by MunifTanjim** [comment, marked as answer]
It's `L`. This is documented in `:help 'statusline'`.
